src/screens/settings/styles.ts

```diff
@@ -56,3 +56,31 @@ export const Description = styled.Text`
   color: ${({ theme }) => theme.colors.BUTTON_DARK_COLOR};
   padding: 0px 16px;
 `;
+
+export const OptionsContainer = styled.View`
+  justify-content: center;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
```

Comment:
remove the position absolute so it doesn't float below the page on smaller screen sizes